### ISSUE TITLE
Notebook output must be ignored when committing

### ISSUE THREAD

**Issue by KrisMinchev** [KrisMinchev/astrobrezi#2]
I believe that it would be beneficial if notebook outputs are not committed. There is a nice gist that promises to solve that issue presented [here](https://gist.github.com/33eyes/431e3d432f73371509d176d0dfb95b6e). It might save bandwidth and space and generally seems to be a good practice to stick to in the future.

**Comment by lyubomirShoylev** [comment]
Sounds good to me! The notebooks still have a lot of work until their finished state, so It might be useful to leave the output in for the time being for development purposes? I am split on the choice. Maybe we can also try to include a minimal example data to: a) help verify the installation is correct; b) be an example of how the package works - what do you think?

There is some additional notebooks we should add from the projects of Nasko and Sasho for the '23 conference: they concern flux calibration by a standard and spectral albedo calculation. I have them on me, but haven't had the time to do it yet.

On the note of installations, maybe it is worth working out a Docker container for the minimal installation in addition to the YAML file for the conda env? This way we can package SpectraPy directly (I think?) as opposed to leaving the manual installation up to the user.

**Comment by KrisMinchev** [comment]
I agree, let's postpone removing notebook output for now. Also including sample data is a great idea, if you have access to some feel free to commit. Same goes for the additional notebooks.

I will look into the idea of creating a Docker container, please open a separate issue whenever you can.
